guide/src/command-line-arguments.md

```diff
@@ -0,0 +1,49 @@
+# Command line arguments
+
+Some configurations can be overwritten using command line agruments.
+
+[[_TOC_]]
+
+## CLI help
+
+To get help on command line options, use `yarner -h`.
+
+```plaintext
+Literate programming compiler
+  https://github.com/mlange-42/yarner
+
+The normal workflow is:
+ 1) Create a project with
+    > yarner create README.md
+ 2) Process the project by running
+    > yarner
+
+USAGE:
+    yarner.exe [FLAGS] [OPTIONS] [input]... [SUBCOMMAND]
```

Comment:
Is that suffix really part of the usage? If so, I think removing it would be preferable both here and there: Windows works without it and it is misleading on other platforms.

Reply:
Ah, yes, it is there because I simply copied to path of `target/debug/yarner.exe` to paste it to the command line. Presence of the suffix depends on whether it is called with or without extension, and clap generates this line accordingly.